scripts/create-deploy-pr

```diff
@@ -2,20 +2,45 @@
 
 set -euo pipefail
 
-ORIGIN=${ORIGIN:-origin}
+GIT_REMOTE=${GIT_REMOTE:-origin}
 SOURCE=${SOURCE:-}
 DEPLOY_BRANCH=stages/prod
 PATCH=${PATCH:-}
 DRY_RUN=${DRY_RUN:-0}
-CHANGELOG_FILE=${CHANGELOG_FILE:-.rc-changelog.md}
+CHANGELOG_FILE=${CHANGELOG_FILE:-tmp/.rc-changelog.md}
+FORMAT_CHANGELOG=${FORMAT_CHANGELOG:-}
+GITHUB_REPOSITORY=${GITHUB_REPOSITORY:-18f/identity-idp}
+STATUS_PROMOTION_LABEL='status - promotion'
 
-function get_last_rc {
-    GH_OUTPUT=$(gh release list --exclude-drafts --exclude-pre-releases --limit 1 || true)
-    if [ -z "$GH_OUTPUT" ]; then
-        echo "Failed to get latest released" >&2
+GH="gh --repo=$GITHUB_REPOSITORY"
```

Comment:
Looks like we could set `$GH_REPO`?

> GH_REPO: specify the GitHub repository in the [HOST/]OWNER/REPO format for commands that otherwise operate on a local repository. [(link)](https://cli.github.com/manual/gh_help_environment)

so maybe we do more like

```suggestion
GH_REPO=${GH_REPO:-18f/identity-idp}
```
and then use plain `gh` in the file?